api/handler/v1/user.go

```diff
@@ -2,33 +2,172 @@ package v1
 
 import (
 	"context"
+	"errors"
+
+	grpczap "github.com/grpc-ecosystem/go-grpc-middleware/logging/zap/ctxzap"
 
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
+	"google.golang.org/protobuf/types/known/structpb"
+	"google.golang.org/protobuf/types/known/timestamppb"
+
+	"github.com/odpf/shield/internal/user"
 
 	shieldv1 "go.buf.build/odpf/gw/odpf/proton/odpf/shield/v1"
 )
 
+type UserService interface {
+	GetUser(ctx context.Context, id string) (user.User, error)
```

Comment:
How about `GetUser` -> `Get`? In the code, we will be using as `UserService.Get()` ?